src/main.spec.js

```diff
@@ -90,5 +90,61 @@ test("Checks HTMLElements", () => {
       );
       expect(child.classList.contains("fs-api-entry")).toBe(true);
     }
+    if (tree.children) {
+      if (tree.previousSibling) {
+        //At first we expect from the li not to contain the "collapse" class, since its contents are diplayed.
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(false);
+        //We click the folder.
+        tree.previousSibling.click();
+        //Now we expect that the list collapses
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(true);
+        //We click the folder.
+        tree.previousSibling.click();
+        //Now we expect for the folders' contents to be displayed.
+        expect(
+          tree.parentNode.classList.contains("fs-api-directory-collapse")
+        ).toBe(false);
+      }
+    }
   }
 });
+
+describe("Tests fetch call", () => {
+  it("mockResponse", async function() {
```

Comment:
We have to rename this as well to something more semantic. We prefer telling in human-readable way what _it_ is being tested here. For example

```javascript
it("Should call `fetch` with the appropriate argument and pass payload to `renderInput`", () => // ...
```